hydra-module.nix

```diff
@@ -273,6 +273,10 @@ in
               runuser -u ${config.services.postgresql.superUser} -- ${config.services.postgresql.package}/bin/createdb -O hydra hydra
               touch ${baseDir}/.db-created
             fi
+            # The extension pg_trgm needs to be created by a superuser.
+            echo "alter user hydra with superuser" | runuser -u ${config.services.postgresql.superUser} -- ${config.services.postgresql.package}/bin/psql
+            echo "create extension pg_trgm" | ${config.services.postgresql.package}/bin/psql -U hydra
```

Comment:
Could this be done via the `postgres` superuser? Temporarily making `hydra` a superuser seems a bit ugly.

Reply:
Yep, this is ugly:/
I added a mapping for the `postgres` user in the `pg_ident.conf`. The `postgres` superuser is now able to create the index for the `hydra` database, so we no longer need to provide superuser capabilities to the `hydra` user.